resources/authgear/templates/en/web/authflowv2/promote.html

```diff
@@ -0,0 +1,167 @@
+{{ template "authflowv2/__page_frame.html" . }}
```

Comment:
Is it possible to share the same template with signup instead of duplicating it?